pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/retention/RetentionManager.java

```diff
@@ -90,7 +90,16 @@ protected void postprocess() {
   }
 
   private void manageRetentionForTable(String tableNameWithType) {
```

Comment:
Let's do the following:

```
  protected void processTable(String tableNameWithType) {
    LOGGER.info("Start managing retention for table: {}", tableNameWithType);
    manageRetentionForTable(tableNameWithType);
    manageSegmentLineageCleanupForTable(tableNameWithType);
  }
```

Let's do this 2 level at the top in the `processTable`. We can think of the retention and segment lineage cleanup as 2 separate steps.

Reply:
I'll prefer skipping adding this printed message since this message will be printed for each of the tables:
```
LOGGER.info("Start managing retention for table: {}", tableNameWithType);
```

Reply:
@jackjlli oh the log is from the original log. Please refer L81.

Reply:
Updated.